docs/guide/single-board-computers.md

```diff
@@ -0,0 +1,39 @@
+# Single Board Computers
+
+## Single Board Computer List
+
+All the commercially available single board computers models included in Webots are included here.
+Users may also add their own custom models.
+Please [contact us](https://www.cyberbotics.com/contact) if you would like to see your favorite single board computer here.
```

Comment:
```suggestion
Please [contact us](https://www.cyberbotics.com/#contact) if you would like to see your favorite single board computer here.
```

Reply:
I would also remove the `www.` prefix from the URL.

Reply:
We have to apply these changes also 4 other pages. I will do these changes in a separate PR.